src/current/cockroachcloud/create-a-serverless-cluster.md

```diff
@@ -33,14 +33,16 @@ You do not need an account with the cloud provider you choose in order to create
 
 ## Step 3. Select the regions
 
-In the **Regions** section, select a region for the cluster. Refer to [{{ site.data.products.db }} Regions](regions.html) for the regions where {{ site.data.products.serverless-plan }} clusters can be deployed.
+In the **Regions** section, select a region for the cluster. Refer to [{{ site.data.products.db }} Regions](regions.html) for the regions where {{ site.data.products.serverless }} clusters can be deployed.
 
 To create a multi-region cluster, click **Add regions** and select additional regions. A cluster can have at most six regions.
 
 For optimal performance, select the cloud provider and region nearest to where your SQL clients, applications, or external data are located. For multi-region clusters, CockroachDB will optimize access to data from the [**Primary region**](../{{site.versions["stable"]}}/multiregion-overview.html). Refer to [Plan a {{ site.data.products.serverless }} Cluster](plan-your-cluster-serverless.html#multi-region-clusters) to learn more. To express interest in additional regions, [contact Support](https://support.cockroachlabs.com) or your Cockroach Labs account team.
 
+After creating a multi-region cluster deployed on AWS, you can optionally [set up AWS PrivateLink](network-authorization.html#aws-privatelink) so that incoming connections to your cluster from applications or services running in your AWS tenant flow over private networks rather than the public internet. For multi-region clusters deployed on GCP, VPC Peering is not yet available.
```

Comment:
```suggestion
After creating a multi-region cluster deployed on AWS, you can optionally [set up AWS PrivateLink](network-authorization.html#aws-privatelink) so that incoming connections to your cluster from applications or services running in your AWS tenant flow over private networks rather than the public internet. For Serverless clusters deployed on GCP, VPC Peering is not yet available.
```